torch/csrc/generic/StorageMethods.cpp

```diff
@@ -267,7 +267,7 @@ static PyObject *THPStorage_(setFromFile)(THPStorage *self, PyObject *args)
   // advanced postion
   const auto fd_current_pos = lseek(fd, 0, SEEK_CUR);
```

Comment:
After reading the MSVC document [here](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/lseek-lseeki64?view=vs-2019), looks like we need to change the function too.